packages/block-editor/src/store/test/actions.js

```diff
@@ -22,7 +22,7 @@ import {
 import * as selectors from '../selectors';
 import reducer from '../reducer';
 import * as actions from '../actions';
-import '../..';
+import { STORE_NAME as blockEditorStoreName } from '../../store/constants';
```

Comment:
It should be enough to use:
```suggestion
import { STORE_NAME as blockEditorStoreName } from '../constants';
```